src/components/Home/README.md

```diff
@@ -0,0 +1,19 @@
+# `<Home />`
+
+This is the base `Home` component provided by [Create React App](https://facebook.github.io/create-react-app/). We've wrapped it up in our prescribed component directory structure which includes:
+
+- `index.css`: component styles
+- `index.tsx`: component code
+- `README.md`: component documentation (hey, that's me!)
+- `screenshots`: visual regression screenshots taken by webdriverio
+- `stories.tsx`: component playground (`npm run test:playground`)
+- `test.tsx`: component tests (`npm run test:unit`)
+
+## Example
+
+![Home](./screenshots/reference/document_320x480.png)
```

Comment:
What purpose does the `document` part of the filename play? Would it be more significant to use the component name in this segment?

Reply:
so this folder is nested inside of the components folder, so it does read src/components/App/screenshots/document_320x480.png. document just refers to that it's a full page screenshot, which I recommend isn't very helpful. I can see both sides of whether it's redundant or not to have src/components/App/screenshots/App_320x480.png, but I want to make sure the redundancy in the screenshot name is shown before implemented.

Reply:
The only reason I think it might be nice to have the component name in the file is for when you might download it locally or it exists on its own. `App_320x480.png` is nicer than `320x480.png` or `document_320x480.png` 🤷‍♂